profiles/kentik_snmp/hp/hpe-switch.yml

```diff
@@ -12,6 +12,7 @@ provider: kentik-switch
 
 sysobjectid:
   - 1.3.6.1.4.1.25506.11.1.*    # HPE Switches
+  - 1.3.6.1.4.1.11.2.3.7.11.*   # http://oid-info.com/get/1.3.6.1.4.1.11.2.3.7.11
```

Comment:
Do we have a reason to believe that HPE switches would use the same OID's as the H3C product lines?  
Going to approve it to allow the user to test but we might need to split these into two profiles based on their feedback.